angular/core/components/discussion_form/discussion_form.coffee

```diff
@@ -2,6 +2,7 @@ angular.module('loomioApp').factory 'DiscussionForm', ->
   templateUrl: 'generated/components/discussion_form/discussion_form.html'
   controller: ($scope, $controller, $location, discussion, Session, Records, AbilityService, FormService, MentionService, AttachmentService, KeyEventService, PrivacyString, EmojiService) ->
     $scope.discussion = discussion.clone()
+    $scope.discussion.groupId = Session.currentGroupId()
```

Comment:
I think I'd move this out into the places where we call
```
ModalService.open DiscussionForm, discussion: Records.discussions.build(Session.groupId)
```

My thoughts there are that could envision us wanting to start a thread that wasn't necessarily the current group's thread; maybe there's a button in the sidebar for each group which says 'start a thread by clicking on me!', and we get stuck because the discussion form itself is locked to the current group.